src/Jobs/Params/ListJobsFilesParameters.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Smartling\Jobs\Params;
+
+use Smartling\Parameters\BaseParameters;
+
+class ListJobFilesParameters extends BaseParameters
+{
+    public function __construct($limit = null, $offset = null)
```

Comment:
In java we decided to make `limit` and `offset` as mandatory to force developer to think about pagination. I propose to do the same in new SDK methods.

Reply:
Fixed